DALI.tex

```diff
@@ -747,8 +747,9 @@ \subsection{Point}
 12.3 45.6
 \end{verbatim}
 
-In spherical coordinates, all longitude values must fall within [0,360] and all
-latitude values within [-90,90].
+All coordinate values must fall within the minimal range of values appropriate for the 
+coordnate system in use. For example, equatorial coordinate limits are [0,360) for longitude 
+and [-90,90] for latitude; other coordinate systems may define different limits.
```

Comment:
yes, technically 360 should be range-reduced to 0 so that's the really correct limit. It did incorrectly say [0,360] previously. For latitude, the extreme values are the poles so both extremes are included in the valid range.

Reply:
That's what I assumed, but I just wanted to be sure. Thanks for the explanation :wink: